src/gui_common/CustomCheckBox.cs

```diff
@@ -27,6 +34,20 @@ private enum CheckState
         Disabled,
     }
 
+    /// <summary>
+    ///   Override base property Group to draw our custom radio icon.
+    /// </summary>
+    public new ButtonGroup? Group
+    {
+        get => base.Group;
+        set
+        {
+            base.Group = value;
+
+            radio = Group != null;
+        }
+    }
```

Comment:
Wouldn't it be more simpler to just check whether it's a radio button by doing something like?
```csharp
public bool Radio => Group != null;
```

Reply:
Makes sense